### ISSUE TITLE
Category widget block: Markup structure inconsistent in editor compared to other widgets

### ISSUE THREAD

**Issue by laurelfulford** [WordPress/gutenberg#12565]
**Describe the bug**
In the editor, the Category widget's list is wrapped in an extra `div`, with the `.wp-block-categories` class on it:

```
<div class="wp-block-categories">
	<ul class="wp-block-categories__list wp-block-categories__list-level-0">
		<li>...</li>
	</ul>
</div>
```

The other widget blocks just have their respective classes (`.wp-block-archives`, `.wp-block-latest-posts`, `.wp-block-latest-comments`) right on the list element:

```
<ul class="wp-block-archives wp-block-archives-list">	
	<li>...</li>
</ul>
```

```
<ul class="wp-block-latest-posts">
	<li>...</li>
</ul>
```

```
<ol class="wp-block-latest-comments has-avatars has-dates has-excerpts">
	<li class="wp-block-latest-comments__comment">...</li>
</ol>
```

This looks to be related to #9411, where this extra div was removed from the front-end.

**To Reproduce**
1. In the editor, add a Category block and an archive block (plus any of the other widget blocks).
2. Examine the markup in the editor.

**Expected behavior**
That the markup structure on the blocks would be similar in the editor. 

**Additional context**
Recreated in Gutenberg 4.6.1, though it's also there in earlier versions.

**Comment by kcalkmc** [comment]
Tested in WP 5.0, Windows 7, Chrome. 
no longer in DIV.

![wp-issue-12565](https://user-images.githubusercontent.com/9540372/49702402-99bf3280-fbef-11e8-81f3-e6a3f1e56b2b.png)

**Comment by Mamaduka** [comment]
The issue was fixed via #43666. Props to @t-hamano!
